src/data/community-events.json

```diff
@@ -16,6 +16,15 @@
     "description": "Preheating activities for DEVCON 7 in the China region.Developers and communities, let us gather at Devcon！",
     "startDate": "2023-10-21",
     "endDate": "2023-10-21"
+  },
+    {
+    "title": "ETH Belgrade",
+    "to": "https://ethbelgrade.rs/",
+    "sponsor": null,
```

Comment:
@minimalsm @corwintines We haven't been enforcing this, so not going to block this PR, but separately we should decide if we want to keep the "sponsor" field going forward or to just remove, since it clearly hasn't been enforced in the current list of events.

Reply:
I created a PR here removing it and typing this data: https://github.com/ethereum/ethereum-org-website/pull/12000

Going to bring this in here, and will fix anything that needs to be done in the above PR.